saos-batch/src/test/java/pl/edu/icm/saos/batch/core/enrichment/TagPostUploadProcessingJobTest.java

```diff
@@ -89,9 +97,14 @@ public void tagPostUploadJob() throws Exception {
         EnrichmentTag scjMaxRefMoneyTag = createEnrichmentTag(testObjectContext.getScJudgmentId(), EnrichmentTagTypes.MAX_REFERENCED_MONEY,
                 JsonNormalizer.normalizeJson("{amount:12300.45, text:'123 tys zł 45 gr'}"));
         
-        enrichmentTagRepository.save(Lists.newArrayList(scjReferenceTag, scjSomeTag, scjMaxRefMoneyTag));
+        LawJournalEntry lawJournalEntry1 = new LawJournalEntry(2014, 34, 344, "Ustawa 1");
+        LawJournalEntry lawJournalEntry2 = new LawJournalEntry(2015, 35, 355, "Ustawa 2");
```

Comment:
It would be nice to have a tag lawJournalEntry that is already in db and to check if it is not overwritten.